common/djangoapps/student/views/dashboard.py

```diff
@@ -19,6 +19,8 @@
 from pytz import UTC
 from six import iteritems, text_type
 
+from edx_django_utils.plugins import get_plugins_view_context
```

Comment:
Nit: Since you fixed the others, I assume you (or I) just missed this one.  Remember to alphabetize if you move it up.